packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html

```diff
@@ -304,6 +304,24 @@ <h3 data-translate="telecom_pack_migration_doing"></h3>
             </div>
         </div>
 
+        <oui-message
+            type="info"
+            data-ng-if="$ctrl.process.selectedOffer.changeTechno"
+        >
+            <p
+                class="mb-0"
+                data-translate="telecom_pack_migration_confirm_temporary_pack_creation_line_1"
+            ></p>
+            <p
+                class="mb-0"
+                data-translate="telecom_pack_migration_confirm_temporary_pack_creation_line_2"
+            ></p>
+            <p
+                class="mb-0"
+                data-translate="telecom_pack_migration_confirm_temporary_pack_creation_line_3"
+            ></p>
```

Comment:
**nitpick**: use ng repeat with [line1, line2, line3] constants defined in constant file

Reply:
Are you kidding? Use a ng-repeat just to display 3 information lines for the customer, for me you want to use a bazooka to kill a fly.

Reply:
Nope, just wanted to compact code and therefore ease code maintenance for the future

Reply:
Nope, just wanted to compact code and therefore ease code maintenance for the future